### ISSUE TITLE
Buildings Often Too Close Together - Collision Volumes Shielding Each Other From Damage Even When Destroyed

### ISSUE THREAD

**Issue by RampantCoyote** [dcs-liberation/dcs_liberation#264]
This is partly a DCS issue - Buildings set too close together, even when visually distinct, can block damage and bomb impacts with their collision volumes even when destroyed. This means one building, even when "dead", can block bomb hits and damage (as far as I can tell). This renders neighboring buildings invulnerable to bombs dropped from certain directions unless they are at very steep angles.

Even "dead" buildings have collision volumes that will absorb bomb hits.

A possible work-around in Liberation would be to space the structures a few meters further apart. Even 2-5 meters of additional spacing could make a difference.

**Comment by DanAlbert** [comment]
Random spawns don't exist any more. If a building template itself is too crowded, we need to know which one.

**Comment by Slowdive1123** [comment]
Dan,

I don't know how exactly to identify the template, but in testing the latest development version I experienced this bug tonight on the Operation Allied Sword Syria Campaign. The site contained:

Oil Derrick x2
Pump Station x2
Subsidiary Structure x1

The structures all appeared ridiculously close and not at the waypoints we had loaded. Nevertheless, we found it visually and landed several 2000lb bomb hits and maverick hits on the structures and none would take damage. Let me know if there's anything I can do to help you track down this issue better.
